src/arch/x86_64/Emit.zig

```diff
@@ -792,34 +409,12 @@ fn dbgAdvancePCAndLine(emit: *Emit, line: u32, column: u32) InnerError!void {
     }
 }
 
-fn mirDbgPrologueEnd(emit: *Emit, inst: Mir.Inst.Index) InnerError!void {
-    _ = inst;
-    switch (emit.debug_output) {
-        .dwarf => |dw| {
-            try dw.setPrologueEnd();
-            log.debug("mirDbgPrologueEnd (line={d}, col={d})", .{
-                emit.prev_di_line,
-                emit.prev_di_column,
-            });
-            try emit.dbgAdvancePCAndLine(emit.prev_di_line, emit.prev_di_column);
-        },
-        .plan9 => {},
-        .none => {},
-    }
-}
+const link = @import("../../link.zig");
```

Comment:
Should the global imports not be at the top of the file? Makes it more readable this way.

Reply:
Actually, you know what, let's leave that in and see how it goes.